doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1276,6 +1281,8 @@ Abbreviations
    Note that abbreviations are not type checked at all, and may result in typing
    errors after expansion.
 
+   This command supports the :attr:`deprecated` attribute.
```

Comment:
I suggest moving this after ```by this name as if it were a proper definition.```  We try to describe the syntax near the beginning of command/tactic descriptions.  Somewhat subjective.